### PR TITLE
Dyn 5340 import export layout

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose
Updating the Import Export layout bearing in mind the longest language (German) as part of the US https://jira.autodesk.com/browse/DYN-5340

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers
@QilongTang 

### FYIs
@filipeotero @RobertGlobant20 @Jingyi-Wen

### CODE REVIEW

**Comment by jesusalvino** [comment]
imagen


imagen

File: src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml [hunk 1]
```diff
@@ -181,7 +181,7 @@
                     Height="14"
                     HorizontalAlignment="Center"
                     VerticalAlignment="Center"
-                    Style="{StaticResource QuestionIcon}"
+                    Style="{StaticResource InformationIcon}"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml:184]: Your screen shot shows a question mark still?

**Review by QilongTang — APPROVED** [review]
Code looks fine but the UI screen shot confused me

File: src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml (hunk 1 above)
Comment (jesusalvino) [reply src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml:184]: > Your screen shot shows a question mark icon still?

it was a naming glitch that I missed, now the NodeAutocomplete uses the InformationIcon and the Preferences uses the QuestionIcon

**Comment by jesusalvino** [comment]
> Code looks fine but the UI screen shot confused me

just displaying the common and longest language as @Jingyi-Wen asked me to check before apply the update

**Comment by jesusalvino** [comment]
> Code looks fine but the UI screen shot confused me

just displaying the common and longest language as @Jingyi-Wen asked me to check before apply the update

File: src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml (hunk 1 above)
Comment (QilongTang) [reply src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml:184]: Gotcha

**Comment by QilongTang** [comment]
LGTM

**Pull request merged**